elasticsearch_dsl/function.py

```diff
@@ -17,9 +17,18 @@
 
 import collections.abc
 from copy import deepcopy
-from typing import Any, ClassVar, Dict, MutableMapping, Optional, Union, overload
+from typing import (
+    Any,
+    ClassVar,
+    Dict,
+    Literal,
+    MutableMapping,
+    Optional,
+    Union,
+    overload,
+)
 
-from .utils import DslBase
```

Comment:
`NotSet` is a class I use to make arguments are not set. `NOT_SET` is the only instance of class `NotSet`. They are used extensively in generated code.

Reply:
Replaced with the transport's `DEFAULT`.